src/settings.h

```diff
@@ -181,7 +181,7 @@ class CClientSettings : public CSettings
 class CServerSettings : public CSettings
 {
 public:
-    CServerSettings ( CServer* pNSerP, const QString& sNFiName ) : CSettings(), pServer ( pNSerP )
+    CServerSettings ( CServer* pNSerP, const QString& sNFiName ) : CSettings(), pServer ( pNSerP ), reSvrInfoIllegal ( QRegExp ( "[!\n;]" ) )
```

Comment:
See comment above for `serverdlg.cpp`. I realise this particular RE is only used for `QString::remove()`, which would imply you do actually intend to remove `!` characters too. (?)